docs/Configuring-Orion/Using-PostgreSQL.md

```diff
@@ -0,0 +1,57 @@
+description: Storing Information with PostgreSQL 
+<!--- END of page meta data -->
+
+# Store Information with PostgreSQL
+
+PostgreSQL is an object-relational database management system you can use to store key values for Orion.
+
+## Prerequisites
+
+* [PostgreSQL](https://www.postgresql.org/)
+
+## Set up your PostgreSQL database
+
+1. In your database, create the `store` table by using the provided DDL script, as shown in the following example: 
+
+    `psql -h HOST-NAME -U USER-NAME -d DATABASE-NAME -f ./docs/Configuring-Orion/database/postgres_storage.sql` 
```

Comment:
Should the ddl file also be in this docs repo? If not we should point out that user needs to access the ddl file from the main Orion repo.

Reply:
Changed to point to the location in the main Orion repo